src/data/pokemon/species_info/gen_1_families.h

```diff
@@ -8359,6 +8432,24 @@ const struct SpeciesInfo gSpeciesInfoGen1[] =
         .iconPalIndex = 2,
         SHADOW(0, 18, SHADOW_SIZE_L)
         FOOTPRINT(Alakazam)
+        #if OW_POKEMON_OBJECT_EVENTS && OW_BATTLE_ONLY_FORMS
+            OVERWORLD(
+                gObjectEventPic_AlakazamMega,
+                SIZE_32x32,
+                SHADOW_SIZE_M,
+                TRACKS_NONE,
+                sAnimTable_Following,
+                gOverworldPalette_AlakazamMega,
+                gShinyOverworldPalette_AlakazamMega
+            )
+            OVERWORLD_FEMALE(
```

Comment:
Female follower data don't need to be set if it's the same as Male data. It defaults to it if not set.